README.md

```diff
@@ -88,14 +88,14 @@ for additional information):
 
     $ sudo apt-get install liboce-foundation-dev liboce-modeling-dev liboce-ocaf-dev liboce-visualization-dev liboce-ocaf-lite-dev
 
-If not, you will need to compile Open CASCADE yourself:
+If not, you will need to compile Open CASCADE yourself (note that the build takes a long time):
 
     $ sudo apt-get install libftgl-dev libtbb2 libtbb-dev libgl1-mesa-dev libfreetype6-dev
     $ git clone https://github.com/tpaviot/oce.git
     $ cd oce
     $ mkdir build && cd build
     $ cmake ..
-    $ make
+    $ make -j
```

Comment:
Hm. interesting. That's exactly what the nix build script tries to determine the hard way.
https://github.com/IfcOpenShell/IfcOpenShell/blob/master/nix/build-all.sh#L64